src/components/cards/BaseStakingCard.sc.tsx

```diff
@@ -88,8 +88,10 @@ export const StakeContainer = styled(Flex)`
 	gap: 6px;
 	width: 100%;
 `;
-export const StakeAmount = styled(Caption)`
-	text-align: center;
+export const StakeAmount = styled(Flex)`
+	align-items: center;
+	justify-content: center;
+	font-size: 0.9em;
```

Comment:
Thanks @divine-comedian 
FYI we have a styled component called FlexCenter that can use instead of this.
also for `font-size: 0.9em` we don't have this in our design system, I prefer this approach:
```tsx
<FlexCenter>
    <StakeAmount></StakeAmount>
    {isLocked && <IconWithTooltip />}
<FlexCenter>
```

Reply:
great! thank you for the suggestion